src/main/java/frc/robot/commands/SetPivotAngle.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.ShooterPivot;
+
+public class SetPivotAngle extends Command {
+  /** Creates a new SetShooterPivot. */
+  ShooterPivot m_shooterPivot;
+  DoubleSupplier m_angleRadians;
+
+  public SetPivotAngle(ShooterPivot shooterPivot, DoubleSupplier angleRadians) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_shooterPivot = shooterPivot;
+    m_angleRadians = angleRadians;
+    addRequirements(m_shooterPivot);
+  }
+
+  public SetPivotAngle(ShooterPivot shooterPivot, double angleRadians) {
+    // Use addRequirements() here to declare subsystem dependencies.
+      this(shooterPivot, () -> angleRadians);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    m_shooterPivot.setAngle(m_angleRadians.getAsDouble());
```

Comment:
you need to save the return from getAsDouble for the isFinished comparison. You don't want to compare against a "moving target".

Reply:
Actually, the following line should be done first so you only have to do the m_angleRadians.getAsDouble() once and you are guaranteed that the angle you are saving in m_wantedAngleRadians is tha value used in m_shooterPivot.serAngle().